<!doctype html>
<html lang="en-US">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
    
        <title>Dmitry Kobak</title>
        <style>
            html{margin:0; padding:0}
            body{font-family: sans-serif; margin:0; padding:0}
            #container{max-width:900px; margin:0 auto; padding:0 25px;}
            header{margin-bottom:3em;}
            h2{font-size:130%}
            h3{font-size:100%}
            li{margin-bottom:0.3em}
            hr{margin-top:2em; height:1px; border:none; color:#555; background-color:#555;}
        </style>
    </head>

    <body>
    <div id="container">
        <h1>Dmitry Kobak</h1>
        
        <header>
        <img src="kobak_sm_round.jpg" alt="dmitry kobak photo" width=150 height=150 
            style="margin:0 50px 10px 25px; float:left;">
            
        <p><a href="mailto:dmitry.kobak@uni-tuebingen.de">dmitry.kobak@uni-tuebingen.de</a>
        
        <p><a href="https://scholar.google.com/citations?user=BUQbD5kAAAAJ"><b>Google Scholar</b></a>
        
        <p>
        <img src="orcid_16x16.png" alt="" width=16 height=16 style="margin-right:3px;margin-bottom:-.1em"><a href="http://orcid.org/0000-0002-5639-7209" style="margin-right:1em">ORCID</a> 
        <img src="github_16x16.png" alt="" width=16 height=16 style="margin-right:3px;margin-bottom:-.1em"><a href="https://github.com/dkobak/" style="margin-right:1em">Github</a>
        <img src="pubmed_16x16.png" alt="" width=16 height=16 style="margin-right:3px;margin-bottom:-.1em"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=dmitry+kobak%5BAuthor%5D&sort=date" style="margin-right:1em">Pubmed</a>
        <img src="dblp_16x16.png" alt="" width=16 height=16 style="margin-right:3px;margin-bottom:-.1em"><a href="https://dblp.org/pid/236/5191" style="margin-right:1em">Dblp</a>
        <img src="publons_16x16.png" alt="" width=16 height=16 style="margin-right:3px;margin-bottom:-.1em"><a href="https://publons.com/researcher/3089678/dmitry-kobak/">Publons</a>
        
        <p>
        <img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-right:5px;margin-bottom:-.2em"><a href="https://twitter.com/hippopedoid">@hippopedoid on Twitter</a><br>
        <img src="crossvalidated_16x16.png" alt="" width=16 height=16 style="margin-right:5px;margin-bottom:-.2em"><a href="https://stats.stackexchange.com/users/28666">@amoeba at CrossValidated.SE</a>
        </header>
        
        <h2 style="color:red">НЕТ ВОЙНЕ | STOP THE WAR</h2>
        
        <p style="clear:both">I am a research scientist in the <a href="http://www.berenslab.org">Berens lab</a> at Tübingen University. I am interested in unsupervised learning, in particular manifold learning and dimensionality reduction for 2D visualization of biological datasets. At the moment I am mostly working with single-cell RNA-seq data coming from the brain. I am also interested in statistical forensics and have been involved in the analysis of Russian electoral falisifications and Covid-19 excess mortality. 
            
        <hr>
        <h2>Teaching</h2>
        <p>In the winter semester 2020/21 I was teaching an introductory course on machine learning (Machine Learning I) for MSc students in neuroscience.
        <ul>
        <li>Videos: <a href="https://www.youtube.com/playlist?list=PL05umP7R6ij35ShKLDqccJSDntugY4FQT">Tübingen Machine Learning / Introduction to Machine Learning</a>
        <li>Slides: 
        <a href="teaching/ml1/kobak_ml1_lecture01_intro_handout.pdf">1</a> | 
        <a href="teaching/ml1/kobak_ml1_lecture02_regression_handout.pdf">2</a> | 
        <a href="teaching/ml1/kobak_ml1_lecture03_likelihood_handout.pdf">3</a> | 
        <a href="teaching/ml1/kobak_ml1_lecture04_regularization_handout.pdf">4</a> | 
        <a href="teaching/ml1/kobak_ml1_lecture05_logistic_handout.pdf">5</a> | 
        <a href="teaching/ml1/kobak_ml1_lecture06_lda_handout.pdf">6</a> | 
        <a href="teaching/ml1/kobak_ml1_lecture07_networks_handout.pdf">7</a> | 
        <a href="teaching/ml1/kobak_ml1_lecture08_boosting_handout.pdf">8</a> | 
        <a href="teaching/ml1/kobak_ml1_lecture09_clustering_handout.pdf">9</a> | 
        <a href="teaching/ml1/kobak_ml1_lecture10_pca_handout.pdf">10</a> | 
        <a href="teaching/ml1/kobak_ml1_lecture11_tsne_noanimations.pdf">11</a>
        </ul>
        
        <hr>
        <h2>Talks</h2>
        <ul>
        <li><a href="https://www.youtube.com/watch?v=CsUqmug7ZMc">Neighbour embeddings for scientific visualization</a>, June 2021, ELLIS Life Heidelberg.
        </ul>
        
        <hr>
        <h2>Supervision</h2>
        PhD students:
        <ul>
            <li><a href="https://github.com/ritagonmar">Rita González Márquez</a> (started in 2022)
            <li><a href="https://github.com/jnboehm">Niklas Böhm</a> (started in 2021)
        </ul>
        MSc students:
        <ul>
            <li><a href="https://github.com/fsvbach">Fynn Bachmann</a> (2021)
            <li><a href="https://github.com/ritagonmar">Rita González Márquez</a> (2021)
            <li><a href="https://github.com/jnboehm">Niklas Böhm</a> (2020)                
        </ul>

        <hr>
        <h2>Recent papers</h2>
        <p>Recent papers are listed below, grouped by topics. I am only listing [co-]first/last author papers here; see my <a href="https://scholar.google.com/citations?user=BUQbD5kAAAAJ">Google Scholar page</a> for the complete list. Most papers are open access; for the ones that are not I provide PDFs. Twitter icons link to the respective Twitter threads (tweeprints). 

        <h3>t-SNE and all that</h3>
        <ul>
            <li><a href="https://openreview.net/forum?id=Hebl3EZ16lq">Two-dimensional visualization of large document libraries using t-SNE</a> (<em>ICLR&nbsp;GTLR&nbsp;workshop</em>,&nbsp;2022)
            <li><a href="https://arxiv.org/abs/2007.08902">A unifying perspective on neighbor embeddings along the attraction-repulsion spectrum</a> (<em>arXiv</em>,&nbsp;2020)&nbsp;<a href="https://twitter.com/hippopedoid/status/1285154214147235840"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>
            <li><a href="https://www.nature.com/articles/s41587-020-00809-z">Initialization is critical for preserving global data structure in both <i>t</i>-SNE and UMAP</a> (<em>Nature&nbsp;Biotechnology</em>,&nbsp;2021;&nbsp;<a href="pdfs/kobak2021initialization.pdf">PDF</a>)&nbsp;<a href="https://twitter.com/hippopedoid/status/1207999178015727616"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>&nbsp;<a href="https://twitter.com/hippopedoid/status/1356286119621636098"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>
            <li><a href="https://link.springer.com/chapter/10.1007/978-3-030-46150-8_8">Heavy-tailed kernels reveal a finer cluster structure in t-SNE visualisations</a> (<em>ECML/PKDD</em>,&nbsp;2020)&nbsp;<a href="https://twitter.com/hippopedoid/status/1263238032360079361"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>
            <li><a href="https://www.nature.com/articles/s41467-019-13056-x">The art of using t-SNE for single-cell transcriptomics</a> (<em>Nature&nbsp;Communications</em>,&nbsp;2019)&nbsp;<a href="https://twitter.com/hippopedoid/status/1206535867831083008"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>
        </ul>
        
        <h3>Statistical methods for transcriptomic and multi-omic data analysis</h3>
        <ul>
            <li><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02451-7">Analytic Pearson residuals for normalization of single-cell RNA-seq UMI data</a> (<em>Genome&nbsp;Biology</em>,&nbsp;2021)&nbsp;<a href="https://twitter.com/hippopedoid/status/1337028792678830084"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>
            <li><a href="https://arxiv.org/abs/2006.10411">Sparse bottleneck networks for exploratory analysis and visualization of neural Patch-seq data</a> (<em>arXiv</em>,&nbsp;2020)
            <li><a href="https://rss.onlinelibrary.wiley.com/doi/10.1111/rssc.12494">Sparse reduced-rank regression for exploratory visualisation of paired multivariate data</a> (<em>Journal&nbsp;of&nbsp;Royal&nbsp;Statistical&nbsp;Society&nbsp;C</em>,&nbsp;2021)&nbsp;<a href="https://twitter.com/hippopedoid/status/1401867282272378884"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>
        </ul>
        
        <h3>Machine learning & statistical theory</h3>
        <ul>
            <li><a href="https://jmlr.org/papers/v21/19-844.html">The optimal ridge penalty for real-world high-dimensional data can be zero or negative due to the implicit ridge regularization</a> (<em>JMLR</em>,&nbsp;2020)&nbsp;<a href="https://twitter.com/hippopedoid/status/1250391223094968321"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>
        </ul>
        
        <h3>Patch-seq data analysis</h3>
        <ul>
            <li><a href="https://www.nature.com/articles/s41586-020-2907-3">Phenotypic variation in transcriptomic cell types in mouse motor cortex</a> (<em>Nature</em>,&nbsp;2020)&nbsp;<a href="https://twitter.com/hippopedoid/status/1224723622268411905"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>
            <li><a href="https://www.nature.com/articles/s41467-019-12058-z">Layer 4 of mouse neocortex differs in cell types and circuit organization between sensory areas</a> (<em>Nature&nbsp;Communications</em>,&nbsp;2019)
        </ul>
        
        <h3>+ Election forensics</h3>
        <ul>
            <li><a href="https://rss.onlinelibrary.wiley.com/doi/full/10.1111/1740-9713.01438">Suspect peaks in Russia's “referendum” results</a> (<em>Significance</em>,&nbsp;2020;&nbsp;<a href="pdfs/kobak2020suspect.pdf">PDF</a>)&nbsp;<a href="https://twitter.com/hippopedoid/status/1315579643106398208"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>
            <li><a href="https://rss.onlinelibrary.wiley.com/doi/full/10.1111/j.1740-9713.2018.01141.x">Putin's peaks: Russian election data revisited</a> (<em>Significance</em>,&nbsp;2018)
            <li><a href="https://rss.onlinelibrary.wiley.com/doi/full/10.1111/j.1740-9713.2016.00936.x">Statistical fingerprints of electoral fraud?</a> (<em>Significance</em>,&nbsp;2016)
            <li><a href="https://projecteuclid.org/euclid.aoas/1458909907">Integer percentages as electoral falsification fingerprints</a> (<em>The&nbsp;Annals&nbsp;of&nbsp;Applied&nbsp;Statistics</em>,&nbsp;2016)
        </ul>
        
        <h3>+ Excess mortality / Covid-19 forensics</h3>
        <ul>
            <li><a href="https://rss.onlinelibrary.wiley.com/doi/10.1111/1740-9713.01627">Underdispersion: A statistical anomaly in reported Covid data</a> (<em>Significance</em>,&nbsp;2022)&nbsp;<a href="https://twitter.com/hippopedoid/status/1361306449952071683"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>&nbsp;<a href="https://twitter.com/hippopedoid/status/1518862282494824448"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>
            <li><a href="https://rss.onlinelibrary.wiley.com/doi/10.1111/1740-9713.01486">Excess mortality reveals Covid's true toll in Russia</a> (<em>Significance</em>,&nbsp;2021)&nbsp;<a href="https://twitter.com/hippopedoid/status/1357615482153631744"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>
            <li><a href="https://elifesciences.org/articles/69336">Tracking excess mortality across countries during the COVID-19 pandemic with the World Mortality Dataset</a> (<em>eLife</em>,&nbsp;2021)&nbsp;<a href="https://twitter.com/hippopedoid/status/1355155183727235073"><img src="twitter_16x16.png" alt="" width=16 height=16 style="margin-bottom:-.2em"></a>
        </ul>
        
        <hr>
        <h2>Previous work and education</h2>
        <p>In 2013&ndash;2016 I was a postdoc in the <a href="http://machenslab.org/">Machens lab</a> at Champalimaud Institute in Lisbon, working on statistical analysis of electrophyisological population recordings from the cortex.
        <ul>
            <li><a href="https://elifesciences.org/articles/44526">State-dependent geometry of population activity in rat auditory cortex</a> (<em>eLife</em>,&nbsp;2019)
            <li><a href="https://elifesciences.org/articles/10989">Demixed principal component analysis of neural population data</a> (<em>eLife</em>,&nbsp;2016)
        </ul>
        
        <p>In 2007&ndash;2012 I did my PhD in the <a href="https://www.brain.uni-freiburg.de/">Mehring lab</a>, initially at Freiburg University and later at Imperial College London, working on computational motor control.
        <ul>
            <li><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00631.2020">Motor skill learning decreases movement variability and increases planning horizon</a> (<em>Journal&nbsp;of&nbsp;Neurophysiology</em>,&nbsp;2022)
            <li><a href="http://www.jneurosci.org/content/32/29/9898.short">Adaptation paths to novel motor tasks are shaped by prior structure learning</a> (<em>The&nbsp;Journal&nbsp;of&nbsp;Neuroscience</em>,&nbsp;2012)
        </ul>
        
        <p>In 2000&ndash;2007 I studied computer science (BSc) and theoretical physics (MSc) in St. Petersburg University.
        
        <p>Before that, I attended St. Petersburg Classical Gymnasium #610. I was part-time teaching there computer science and physics in 2002&ndash;2006 while studying in university. In 2004, together with a friend, I made a website <a href="http://610.ru">610.ru</a> that is <i>still</i> online (with minor changes). 
        
        <p style="margin-top:3em; font-size:80%; text-align:right; font-style:italic">Last updated: May 2, 2022
        
        <div><img src="kobaks1.jpg" width=100% alt="kobaks" style="display:block; margin:1em 0"></div> 
    </div>
    </body>
</html>

